client/utils/utils.c

```diff
@@ -25,6 +25,7 @@
 #include "log.h"
 #include "../system.h"
 #include "math.h"
```

Comment:
I think it makes sense to rename our internal math.h to something like xdag_math.h.
Not related to the current PR. I can do it

Reply:
Agree. I will try in next PR.